Circles_intersection.py

```diff
@@ -0,0 +1,28 @@
+import math
```

Comment:
It is the wrong location. The file should have been in `homework/12_V/16_Kayra/homework01/`. Anyway I will move the files to the correct dir.